dev/devicelab/README.md

```diff
@@ -7,15 +7,79 @@ This package contains the code for test framework and the tests. More generally
 the tests are referred to as "tasks" in the API, but since we primarily use it
 for testing, this document refers to them as "tests".
 
-If you have access to Google's internal network, you can see the continuous
-build results from the master branch at <http://go/flutter-dashboard/build.html>.
-(There is currently no public view of this data, unfortunately.)
+Build results are available at https://flutter-dashboard.appspot.com.
 
-# Prerequisites
+# Reading the dashboard
 
-You must set the `ANDROID_HOME` environment variable to run tests on Android. If
-you have a local build of the Flutter engine, then you have a copy of the
-Android SDK at `.../engine/src/third_party/android_tools/sdk`.
+## The build page
+
+The build page is accessible at https://flutter-dashboard.appspot.com/build.html.
+This page reports the health of build servers, called _agents_, and the statuses
+of build tasks.
+
+### Agents
+
+A green agent is considered healthy and ready to receive new tasks to build. A
+red agent is broken and does not receive new tasks.
+
+In the example below, the dashboard shows that the `linux2` agent is broken and
+requires attention. All other agents are healthy.
+
+![Agent statuses](images/agent-statuses.png)
+
+### Tasks
+
+The table below the agent statuses displays the statuses of build tasks. Task
+statuses are color-coded. The following statuses are available:
+
+**New task** (light blue): the task is waiting for an agent to pick it up start
+the build.
+
+**Task is running** (spinning blue): an agent is currently building the task.
+
+**Task succeeded** (green): an agent reported a successful completion of the
+task.
+
+**Task is flaky** (yellow): the task was attempted multiple time, but only the
```

Comment:
Does the task always run twice, or only a second time if it the first one failed?

Reply:
Good question. Added "How the devicelab runs the tasks" section.